api/types/types.proto

```diff
@@ -1154,15 +1150,48 @@ message LocalAuthSecrets {
     // PasswordHash encodes a combined salt & hash for password verification.
     bytes PasswordHash = 1 [ (gogoproto.jsontag) = "password_hash,omitempty" ];
 
-    // TOTPKey is the key used for Time-based One Time Password varification.
+    // Deprecated 2nd factor fields, use MFA below instead.
     string TOTPKey = 2 [ (gogoproto.jsontag) = "totp_key,omitempty" ];
-
-    // U2FRegistration holds Universal Second Factor registration info.
     U2FRegistrationData U2FRegistration = 3 [ (gogoproto.jsontag) = "u2f_registration,omitempty" ];
-
-    // U2FCounter holds the highest seen Universal Second Factor registration
```

Comment:
Did you remove godocs on purpose?

Reply:
Yes, these 3 fields are deprecated and only exist to allow the auth server to load old data from the backend and migrate it to `MFA`